libs/components/tabs/testing/src/tabs/tab-fixture-tab.ts

```diff
@@ -4,12 +4,12 @@
  */
 export interface SkyTabsetFixtureTab {
   /**
-   * Indicates whether the tab is the currently selected tab.
+   * Whether the tab is the currently selected tab.
```

Comment:
```suggestion
   * Whether the tab is selected.
```
Not related to the other changes, but it seems like we could tighten this up.